src/Core/Components/List/FluentAutocomplete.razor.cs

```diff
@@ -127,6 +127,18 @@ public override bool Multiple
     [Parameter]
     public RenderFragment<IEnumerable<TOption>>? FooterContent { get; set; }
 
+    /// <summary>
+    /// Title and Aria-Label for the Scroll to previous button.
+    /// </summary>
+    [Parameter]
+    public string TitleScrollToPrevious { get; set; } = "Scroll to previous";
+
+    /// <summary>
+    /// Title and Aria-Label for the Scroll to next button.
+    /// </summary>
+    [Parameter]
+    public string TitleScrollToNext { get; set; } = "Scroll to next";
```

Comment:
I would just set default for these to 'Previous' and 'Next'

Reply:
Why not... Fixed